src/cas_cli/client.py

```diff
@@ -209,7 +220,35 @@ def annotate_anndata(
         self._print("Finished!")
         return result
 
-    def annotate_10x_h5(
+    def annotate_anndata_file(
+        self,
+        filepath: str,
+        chunk_size=2000,
+        feature_schema_name: str = "default",
+        count_matrix_name: str = "X",
+        feature_ids_column_name: str = "index",
+    ) -> t.List[t.Dict[str, t.Any]]:
+        """
+        Read 10x `anndata.AnnData` matrix and apply the annotate method on it.
```

Comment:
It's not 10x `anndata.AnnData`, it's just `anndata.AnnData` :D

Reply:
Omg, right...